src/ovs/aca_ovs_l2_programmer.cpp

```diff
@@ -640,28 +641,28 @@ void ACA_OVS_L2_Programmer::execute_openflow(ulong &culminative_time,
                                              const std::string flow_string,
                                              const std::string action)
 {
-  ACA_LOG_DEBUG("%s", "ACA_OVS_L2_Programmer::execute_openflow ---> Entering\n");
-  auto openflow_client_start = chrono::steady_clock::now();
+  // ACA_LOG_DEBUG("%s", "ACA_OVS_L2_Programmer::execute_openflow ---> Entering\n");
+  // auto openflow_client_start = chrono::steady_clock::now();
```

Comment:
if it doesn't impact performance as I remember, we can turn it back on

Reply:
Done.